RAGchain/benchmark/dataset/msmarco.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, run_pipeline: BasePipeline,
 
         self.eval_size = evaluate_size
         self.run_pipeline = run_pipeline
-        self.data = self.dataset['test']
```

Comment:
Why not using test...? For testing

Reply:
@vkehfdl1 Because MSMARCO dataset v2.1 passages' 'is_selected' values  are all -1 like below picture. So I use train data. I validate this using below code.

```python
count = 0
for i in range(len(passage)):
    
    for n in passage[i]['is_selected']:
        if n != -1:
            print('값: ', n)
            print('passage: ', passage[i])
            count += 1

```

![image](https://github.com/NomaDamas/RAGchain/assets/60510718/de334a5e-6633-4791-a634-7e8c03e3337a)